modules/32_power/IntC/datainput.gms

```diff
@@ -44,6 +44,7 @@ f32_factorStorage(all_regi,"windoff") = f32_factorStorage(all_regi,"wind");
 f32_factorStorage(all_regi,"wind")      = 1.35 * f32_factorStorage(all_regi,"wind"); 
 $ENDIF.WindOff
 p32_factorStorage(all_regi,all_te) = f32_factorStorage(all_regi,all_te);
+p32_factorStorage(all_regi,"spv") = f32_factorStorage(all_regi,"spv") *1.3;
```

Comment:
Why 1.3? is it to account for pv's lower capacity factor than wind? but then why does wind get 1.35 as well?

Reply:
ah, I'll remove this, this was left over from my trying to adjust spv downward due to CA request (from IKEA)